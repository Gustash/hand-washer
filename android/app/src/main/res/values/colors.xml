<resources>
  <color name="white">#00A778</color>
</resources>
