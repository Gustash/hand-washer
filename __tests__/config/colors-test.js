import colors from '../../app/config/colors'

describe('colors config', () => {
  it('matches snapshot', () => {
    expect(JSON.stringify(colors)).toMatchSnapshot()
  })
})
