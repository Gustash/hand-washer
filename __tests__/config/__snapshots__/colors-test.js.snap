// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors config matches snapshot 1`] = `"{\\"lightBlue\\":\\"#00A1D2\\",\\"blue\\":\\"#0000FF\\",\\"green\\":\\"#00A778\\",\\"white\\":\\"#FFFFFF\\",\\"black\\":\\"#000000\\",\\"darkGrey\\":\\"#4A4A4A\\",\\"lightGrey\\":\\"#9B9B9B\\",\\"separator\\":\\"#E6E6E6\\",\\"brandedText\\":\\"#1C3A6A\\"}"`;
